### PR TITLE
Updating cli docs for octo.exe

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the command line docs for octo.exe.

Created by TeamCity project 'Automated Documentation Updates::Update CLI Docs', build 187

### CODE REVIEW

File: docs/octopus-rest-api/octopus-cli/install-autocomplete.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: install-autocomplete
+description: Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash & powershell.
+---
+
+Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash & powershell.
+
+**install-autocomplete options**
+
+```text
+Please specify the type of shell to install autocompletion for: --shell=XYZ. Valid values are Pwsh, Zsh, Bash and Powershell.
```
Comment (matt-richardson) [inline docs/octopus-rest-api/octopus-cli/install-autocomplete.md]: @jburger - looks like we missed a test - the command doesn't support help properly.

File: docs/octopus-rest-api/octopus-cli/complete.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+title: complete
+description: Supports command line auto completion.
+---
+
+Supports command line auto completion.
+
+**complete options**
+
+```text
+--help
```
Comment (matt-richardson) [inline docs/octopus-rest-api/octopus-cli/complete.md]: @jburger - this is also a bit weird...

File: docs/octopus-rest-api/octopus-cli/install-autocomplete.md (hunk 1 above)
Comment (jburger) [reply docs/octopus-rest-api/octopus-cli/install-autocomplete.md]: @matt-richardson thanks I'll see about another PR to the CLI to fix this up

**Comment by matt-richardson** [comment]
Superseded by #662

**Pull request closed without merging**